easybuild/easyconfigs/a/Alpha/Alpha-20190607-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Bundle'
+
+name = 'Alpha'
+version = '20190607'
+local_alpha_commit = 'e027eb0fea1e'
+local_pyinc_commit = '0f8a00b8c10c'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.lirmm.fr/~swenson/alpha/alpha.htm'
+description = "Alpha is a tool designed for detailed comparative study of bacteriophage genomes."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAST+', '2.9.0'),
+    ('Clustal-Omega', '1.2.4'),
+    ('GenomeTools', '1.6.1'),
+    ('GTK+', '3.24.13'),
+    ('matplotlib', '2.2.4', versionsuffix),
+    ('networkx', '2.2', versionsuffix),
+    ('pydot', '1.4.1'),
+    ('PyGObject', '3.34.0', versionsuffix),
+    ('pygraphviz', '1.5'),
+    ('python-igraph', '0.8.0'),
+]
+
+components = [
+    ('progressbar', '2.5', {
+        'easyblock': 'PythonPackage',
+        'source_urls': [PYPI_SOURCE],
+        'sources': ['%(name)s-%(version)s.tar.gz'],
+        'checksums': ['5d81cb529da2e223b53962afd6c8ca0f05c6670e40309a7219eacc36af9b6c63'],
+        'start_dir': '%(name)s-%(version)s',
+    }),
+    ('pyinclude', '20190223', {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://bitbucket.org/thekswenson/%(name)s/get/'],
+        'sources': [{'download_filename': '%s.zip' % local_pyinc_commit, 'filename': '%(name)s-%(version)s.zip'}],
+        'checksums': ['9dc9b2bafbfe7a694ff4025164dc6e356989ee89b914691457fb0a474fe66c9e'],
+        'start_dir': 'thekswenson-%%(name)s-%s' % local_pyinc_commit,
+        'install_type': 'subdir',
+    }),
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://bitbucket.org/thekswenson/%(namelower)s/get/'],
+        'sources': [{'download_filename': '%s.zip' % local_alpha_commit, 'filename': '%(namelower)s-%(version)s.zip'}],
```

Comment:
@lexming Hmm, download is failing for this?

```
== 2020-04-30 10:38:32,651 filetools.py:544 DEBUG Trying to download alpha-20190607.zip from https://bitbucket.org/thekswenson/alpha/get/e027eb0fea1e.zip to /sources/a/Alpha/alpha-20190607.zip
== 2020-04-30 10:38:32,651 filetools.py:551 DEBUG Using timeout of 10 seconds for initiating download                                  
== 2020-04-30 10:38:32,651 filetools.py:1493 DEBUG Not creating existing path /sources/a/Alpha
== 2020-04-30 10:38:33,129 filetools.py:594 WARNING URL https://bitbucket.org/thekswenson/alpha/get/e027eb0fea1e.zip was not found (HTTP response code 404), not trying again
== 2020-04-30 10:38:33,129 filetools.py:620 WARNING Download of https://bitbucket.org/thekswenson/alpha/get/e027eb0fea1e.zip to /sources/a/Alpha/alpha-20190607.zip failed, done trying
```

Reply:
The developer deleted that commit! WTF! /facepalm

Reply:
fixed :+1:

Reply:
Oh nice... Maybe reach out to them and ask wtf is going on?